browser/extensions/api/brave_shields_api.cc

```diff
@@ -111,6 +128,8 @@ ExtensionFunction::ResponseAction BraveShieldsSetAdControlTypeFunction::Run() {
   Profile* profile = Profile::FromBrowserContext(browser_context());
   ::brave_shields::SetAdControlType(profile, control_type, url);
 
+  RecordShieldsSettingChanged();
```

Comment:
this is definitely not the right place to put this because it won't be captured on android. I think it should go in `brave_shields::SetAdControlType`

Reply:
fixed